src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -546,6 +568,18 @@ private boolean areSomeSelectedChatsUnpinned() {
     return false;
   }
 
+  private boolean areSomeSelectedChatsUnmuted() {
```

Comment:
i would have found the code easier to read if this function would be `areSomeSelectedChatsMuted()` - but i see that this double-inverting is already present with `areSomeSelectedChatsUnpinned`, so, no blocker :)